llama-index-integrations/readers/llama-index-readers-slack/llama_index/readers/slack/base.py

```diff
@@ -61,9 +66,17 @@ def __init__(
                 "variable `SLACK_BOT_TOKEN`."
             )
         if ssl is None:
-            self._client = WebClient(token=slack_token)
+            self._client = (
```

Comment:
imo we should just load both a `client` and `aclient` - no need to toggle with a boolean, both should be loaded. The code paths are completely separate no?

Reply:
make sense!